worknenjoy/gitpay
# Feature/issue 514

## Description

> Adds a link to add the deadline as a Google Calendar event to the "Assigned" date

## Issue

> Closes #514

## Impacted Area

> Mail

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

> * Create a user
> * Import an issue
> * Set deadline
> * Create another user
> * Apply to solve that issue
> * Log with the first user
> * Assign that user
> * The email notification should have the link to add to the calendar



## Before
> Screenshot from the state before
![image](https://user-images.githubusercontent.com/25051063/78732788-a05bd800-7911-11ea-8916-f8cc507a827c.png)


## After
> Screenshot from the state after your Pull Request
![image](https://user-images.githubusercontent.com/25051063/78732805-ad78c700-7911-11ea-9488-22035af04877.png)
